src/ElementScreenTaker.js

```diff
@@ -4,55 +4,71 @@ let captureWebsite = require('capture-website');
 let puppeteer = require('puppeteer');
 let cheerio = require('cheerio');
 let id = [];
-let url = process.cwd()+`\\src\\test.html`;
+let url = process.cwd() + `\\src\\test.html`;
 
 export default class ElementScreenTaker {
     constructor(value) {
-
     }
 
 
     // takes a screen shot from each newsarticle on the generated html page'
     //test: returns image Path
     //aantal plaatjes maken via varible
     async takeScreenShots() {
-        try{
-            await this.getCssSelectors();
+        try {
+            await this.getCssSelectors(process.env.selector);
             let time = new Date();
-            let t = time.getDate()+"_"+time.getMonth()+"_"+time.getFullYear()+"_"+time.getHours()+"_"+time.getMinutes();
+            let t = time.getDate() + "_" + time.getMonth() + "_" + time.getFullYear() + "_" + time.getHours() + "_" + time.getMinutes();
             for (let index = 0; index < id.length; index++) {
-                    await captureWebsite.file(url, `${process.env.IMAGE_PATH+'//'+t+'_'+index}.png`, { waitForElement: `#${id[index]}`, element: `#${id[index]}`, hideElements: [
-                    `#${id[index-1]}`,
-                    `#${id[index+1]}`
-                ]  });
+                await this.captureScreens(t,index);
             }
-            
-            return `${process.env.IMAGE_PATH+'//'+t+'_'}`;
-        }catch(err){
+            return `${process.env.IMAGE_PATH + '//' + t + '_'}`;
```

Comment:
volgens mij ben je hier een beetje string interpolation aan het mixen met programmatuur, ik denk dat alles in interpolation kan